notebooks/EDA Climate.ipynb

```diff
@@ -0,0 +1,1821 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    seasonal_decompose(df_ocean+10, model='multiplicative', period=4).resid.plot()
</pre><p>Plot all three components when you do a decomposition</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jessegrabowski/laos-climate-change/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>